addons/resource/views/resource_calendar_views.xml

```diff
@@ -54,13 +54,12 @@
                         <group>
                             <field name="active" invisible="1"/>
                             <field name="company_id" groups="base.group_multi_company"/>
-                            <field name="hours_per_day" widget="float_time"/>
-                            <field name="tz" widget="timezone_mismatch" options="{'tz_offset_field': 'tz_offset', 'mismatch_title': 'Timezone Mismatch : This timezone is different from that of your browser.\nPlease, be mindful of this when setting the working hours or the time off.'}" />
-                            <field name="tz_offset" invisible="1"/>
+                            <field name="hours_per_day" widget="float_time" readonly="not flexible_hours"/>
+                            <field name="flexible_hours"/>
```

Comment:
To be discussed with AJU, but shouldn't the flexible hours field be above the field that it makes invisible rather than below? I feel like having it below will make the interface confusing

Reply:
Thanks for the feedback, I will forward the message to AJU !

Reply:
@Bertrand2 
I incorporate the feedback/changes discussed on the Task discussion and applied the changes.